genesys:
  adma_ros2_driver:
    ros__parameters:
      destination_ip: '127.0.0.1'
      destination_port: 1040
      use_performance_check: False
      protocol_version: 'v3.3.4'
      frame_ids:
        navsatfix: 'gnss_link'
        imu: 'imu_link'
        adma: 'adma'
        adma_status: 'adma_status'
        raw_data: 'data_raw'
        odometry_pose_id: 'adma' # header.frame_id (basicly could be a "name" of desired POI)
        odometry_twist_id: 'odometry' # odometry.child_frame_id
      odometry_yaw_offset: 0.0 # in degrees
  gsdb_server:
    ros__parameters:
      ip_address: '127.0.0.1'
      port: 1040
      frequency: 100
      protocol_version: 'v3.3.4'
      gsdb_file: '/home/$USER/$ROS2_WS/data/raw_data.gsdb'